github-pullrequest-plugin/src/main/java/com/github/kostyasha/github/integration/branch/events/impl/commitchecks/GitHubBranchCommitCheck.java

```diff
@@ -30,4 +31,16 @@ public GitHubBranchCommitCheckDescriptor getDescriptor() {
      * @return <code>GitHubBranchCause</code> instance indicating if the build should be skipped, <code>null</code> otherwise.
```

Comment:
mmm...does it follow cause.isSkip() or you hardcoded logic to do skip aka "prevent"?